python/acquire/__init__.py

```diff
@@ -102,33 +111,21 @@ def setup_two_streams(runtime: Runtime, frame_count: int) -> Properties:
     return p
 
 
-g_runtime = None
-
-
 def gui(
     viewer: "napari.Viewer",
     frame_count: int = 100,
-):
+    stream_count: Literal[1, 2] = 2,
```

Comment:
I added this argument to allow exercising both the 1 stream (after fixing it) and 2 stream setup code. Alternatively, we could just remove the 1 stream setup.